src/main/kotlin/uk/gov/justice/digital/hmpps/educationandworkplanapi/app/resource/GoalController.kt

```diff
@@ -133,16 +133,14 @@ class GoalController(
     @Valid @RequestBody archiveGoalRequest: UnarchiveGoalRequest,
     @PathVariable @Pattern(regexp = PRISON_NUMBER_FORMAT) prisonNumber: String,
     @PathVariable goalReference: UUID,
-  ): Goal {
```

Comment:
Minor tidying - giving the method is annotated with `@ResponseStatus(HttpStatus.NO_CONTENT)` and the swagger spec defines it as returning no body, the function shouldn't really return anything 👍

Reply:
Should this be the same with Archive Goal?

Reply:
Good shout; yes, I've just pushed a commit that does that too 👍